### DISCUSSION TITLE
Table row middle-clickable

### DISCUSSION THREAD

**Discussion by sasgosh** [quasarframework/quasar#10447] [General - Components / Directives / etc]
Hello,

How to make entire table row clickable with middle button? I am using @row-click method, I didn't find any other way, but it is activated only on the left click while the middle one does not work.

Actually, I just need to go to the link and nothing more. Now I have solved it with:

`@row-click="(ev, row) => {$router.push(url + row.id)}"`

but if there is a better solution, which includes a middle click, then it would be very nice.

Thanks

**Comment by MatthewSH** [comment]
What do you mean by "middle button"? Are you referring to the scroll wheel?
Have you looked into the `@mouseDown.middle` event? Just created a simple example [here](https://codesandbox.io/embed/youthful-andras-onrv6?fontsize=14&hidenavigation=1&theme=dark), just middle click the number and it should increment.

EDIT: I missed the mark: https://github.com/quasarframework/quasar/discussions/10447#discussioncomment-1223874

**Reply by sasgosh** [reply]
Well, I thought opening page in new tab with middle button. Just like @row-click does but in new tab. @mouseDown.middle looks interesting but how to get current row id?

**Reply by MatthewSH** [reply]
You know what, that's on me. I completely missed something there. If I may ask, what's the value of having a middle-click event? If there's real value I could probably create a PR for adding it.

**Reply by sasgosh** [reply]
Well, if you want to open link to new tab, you use middle click. It doesn't have to be middle-click event, it is enough to modify @row-click event so can be run with middle click, or (better) add new property :row-to which is like router-link 'to' property.

**Reply by MatthewSH** [reply]
`@row-click` event is fired on the actual left mouse click, you also have double click and context menu. There is no middle-click event fired from the table.
However, you make a good point. When middle-clicking links they do open a new tab so re-creating that behaviour should be possible. I'll make a PR and get feedback to see what other people have to say.

**Reply by MatthewSH** [reply]
Created PR #10460 for this, will see what members say.
